### PR TITLE
lrzip: add page

### PR DESCRIPTION
**Pull request by russsaidwords**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

**Comment by sbrl** [comment]
I'd agree with @pxgamer  here. You can always add something like

```markdown
> See also `lrztar`, `lrzuntar`, and `lrunzip`.
```

to the description.

**Comment by russsaidwords** [comment]
I'll work on that later today. Do I create new PRs for the 3 new pages or do I add them here as a commit?

**Comment by sbrl** [comment]
I'd create a separate PR for each page.

**Comment by russsaidwords** [comment]
Okay, can do.

File: pages/linux/lrzip.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# lrzip
+
+> A large file compression program. See also `lrunzip`, `lrztar`, `lrzuntar`.
```
Comment (pxgamer) [inline pages/linux/lrzip.md]: Maybe put the `See also` sentence on the line below?

**Pull request merged**